wavefront_pyformance/delta.py

```diff
@@ -62,5 +62,8 @@ class DeltaCounter(meters.Counter):
     every time the value is reported.
     """
 
-    DELTA_PREFIX = '\u2206'
-    ALT_DELTA_PREFIX = '\u0394'
+    DELTA_PREFIX = '∆'  # u"\u2206"
```

Comment:
It might be worth leaving these as codes instead of literals, to make sure you have different delta symbols here, as literals look really similar